packages/ifpack2/src/Ifpack2_BlockTriDiContainer_impl.hpp

```diff
@@ -107,6 +109,12 @@ namespace Ifpack2 {
     template <typename ViewType>
     using ConstUnmanaged = Const<Unmanaged<ViewType> >;    
```

Comment:
These modifying aliases look like things we could promote to live in kokkos-kernels or even Kokkos at some point.